### PR TITLE
Feature/contacts section

### PR DESCRIPTION
**Pull request by min-in-zon**

### CODE REVIEW

File: src/components/common/icons/Facebook.jsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+const FacebookIcon = () => {
```
Comment (adadychanka) [inline src/components/common/icons/Facebook.jsx:1]: https://www.freecodecamp.org/news/how-to-import-svgs-in-react-and-vite/

Let's use 6th approach to handle svgs

File: src/pages/MainPage/ContactsSection.jsx [hunk 2]
```diff
@@ -0,0 +1,88 @@
+import './contactsSection.scss';
+import mailIcon from '../../assets/img/mailIcon.svg';
+import FacebookIcon from '../../components/common/icons/Facebook';
+import instaIcon from '../../assets/img/instaIcon.svg';
```
Comment (adadychanka) [inline src/pages/MainPage/ContactsSection.jsx:4]: Why don't you update other icons as well?

File: src/pages/MainPage/ContactsSection.jsx [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+                placeholder="Your email"
+                required
+              />
+              <textarea
+                name="message"
+                rows="5"
+                placeholder="Your message"
+                required
+              ></textarea>
+              <button type="submit">Send Message</button>
```
Comment (adadychanka) [inline src/pages/MainPage/ContactsSection.jsx:79]: Are you going to implement email-sending functionality in the PR or the following one?

File: src/pages/MainPage/contactsSection.scss [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+    &-form {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+
+        h3 {
+            margin-bottom: 0.625rem;
+        }
+
+        form {
```
Comment (adadychanka) [inline src/pages/MainPage/contactsSection.scss:104]: it's not really a good practice to stick to tags in styles, it's better to create class names for that

**Review by adadychanka — CHANGES_REQUESTED** [review]

File: src/pages/MainPage/ContactsSection.jsx (hunk 3 above)
Comment (min-in-zon) [reply src/pages/MainPage/ContactsSection.jsx:79]: I'm going to add all the functionality after creating all html and switching it to tailwind.сss

**Review by adadychanka — APPROVED** [review]

**Pull request merged**